core/schema/src/main/liquibase/28.0.0/changelog.xml

```diff
@@ -474,7 +477,295 @@
         <constraints nullable="false"/>
       </column>
     </createTable>
-    <addPrimaryKey tableName="bmp_route_info" constraintName="pk_bmp_route_info" columnNames="prefix, prefix_len, origin_as"/>
+    <addPrimaryKey tableName="bmp_route_info" constraintName="pk_bmp_route_info"
+                   columnNames="prefix, prefix_len, origin_as"/>
+  </changeSet>
+
+
+  <changeSet runOnChange="false" author="cgorantla" id="28.0.0-bmp-ip-rib-log-nxtid">
+    <preConditions onFail="MARK_RAN">
+      <sqlCheck expectedResult="0">
+        SELECT COUNT(relname) AS SEQUENCE_NAME FROM pg_class, pg_namespace
+        WHERE relkind='S' AND pg_class.relnamespace = pg_namespace.oid AND relname ILIKE 'bmpipriblognxtid'
+      </sqlCheck>
+    </preConditions>
+    <createSequence sequenceName="bmpipriblognxtid"/>
+    <rollback>
+      <dropSequence sequenceName="bmpipriblognxtid"/>
+    </rollback>
+  </changeSet>
+
+  <changeSet author="cgorantla" id="27.0.0-bmp-ip-rib-log">
```

Comment:
27.0.0 for the id? This is the 28.0.0 changelog, so I think it should contain only id starting with 28.